docs/chapters/baseplatform/replacing-packagegroups.rst

```diff
@@ -0,0 +1,57 @@
+:orphan:
+
+.. _replacing-packagegroups:
+
+Replacing package groups by hand-picked components
+==================================================
+
+By default PELUX include components from QT Auto using the package groups feature.
+This chapter describes how to replace these default QT Auto components with the
+handpicked components.
+
+Following package groups from Qt Auto are included in the PELUX image. These package groups
+add several QT Auto add-ons (includes ``neptune-ui`` among others) and packages that are used 
+when building SDK for host and target machines.
+
+* ``packagegroup-b2qt-automotive-addons``
+* ``nativesdk-packagegroup-b2qt-automotive-qt5-toolchain-host``
+* ``packagegroup-b2qt-automotive-qt5-toolchain-target``
+
+``packagegroup-b2qt-automotive-addons`` includes ``neptune-ui`` component among others which is 
+a reference Qt Auto UI. However, in case a specific distribution do not want to use Qt Auto 
+components in this package group or it only want to use a subset of Qt Auto components, then
+it is possible to replace the package group with hand-picked components. 
+
+To replace for example, ``packagegroup-b2qt-automotive-addons`` with only ``neptune-ui`` 
+component, update the file ``meta-pelux/layers/b2qt/recipes-core/images/core-image-pelux-qtauto-neptune.bb``
```

Comment:
The common way of changing what's happening is through appends. We should recommend using a bbappend instead of changing the base recipe.

Reply:
Yes that makes sense, since it preserves the recommended layered approach. Will fix it.